parm/config/gefs/config.base

```diff
@@ -290,19 +299,10 @@ export NMEM_ENS=@NMEM_ENS@
 export ENSMEM=${ENSMEM:-"000"}
 export MEMDIR="mem${ENSMEM}"
 
-# initialize ocean ensemble members with perturbations
-#   if true, only occurs for members greater than zero
-export REPLAY_ICS=@REPLAY_ICS@
-if [[ "${REPLAY_ICS:-NO}" == "YES" ]]; then
-    export OFFSET_START_HOUR=$(( assim_freq / 2 ))
-else
-    export OFFSET_START_HOUR=0
-fi
-
 export DOIAU="NO"  # While we are not doing IAU, we may want to warm start w/ IAU in the future
 # Check if cycle is cold starting
 if [[ "${EXP_WARM_START}" = ".false." ]]; then
-  export IAU_FHROT=${OFFSET_START_HOUR}
+    export OFFSET_START_HOUR=$(( assim_freq / 2 ))
```

Comment:
It looks like all CI tests passed including `C96_S2SWA_gefs_replay_ics`, but `C48_S2SWA_gefs` failed. After reading the logs, it looks like `OFFSET_START_HOUR` is set to `3` here for `C48_S2SWA_gefs`. Shouldn't `OFFSET_START_HOUR` be `0` for non-replay GEFS?

Reply:
Thanks @NeilBarton-NOAA. After a quick test on WCOSS2, reverting the change on line 305 resolved the issue that occurred in the `C48_S2SWA_gefs` CI test. `OFFSET_START_HOUR` is already set earlier in `config.base`.